anemometer/app/price-feed.js

```diff
@@ -0,0 +1,59 @@
+const ethers  = require('ethers');
+const fs      = require('fs');
+const axios = require('axios');
+
+ 
+const root                = 'iexec_out';
+const determinismFilePath = `${root}/determinism.iexec`;
+const callbackFilePath    = `${root}/callback.iexec`;
+const errorFilePath       = `${root}/error.iexec`;
+
+/*****************************************************************************
+ *                                  CONFIG                                   *
+ *****************************************************************************/
+const APIKEY = '87a1fd74e099f170b76906e87e19e9d9';
+
+/*****************************************************************************
+ *                                 ARGUMENTS                                 *
+ *****************************************************************************/
+
+var [ lati , longi] = process.argv.slice(2).map(s => s);
+console.log(longi);
+console.log(lati);
+/*****************************************************************************
+ *                                HTTP QUERY                                 *
+ *****************************************************************************/
+
+let path = `https://api.openweathermap.org/data/2.5/weather?lat=${lati}&lon=${longi}&APPID=${APIKEY}`;
+
+/*****************************************************************************
+ *                                  EXECUTE                                  *
+ *****************************************************************************/
+axios.get(path)
+.then(res=>{
+	var result = res.data;
+	var windSpeed = result.wind.speed;
+	var weather =  result.weather[0].description;
+	var timestamp = new Date().getTime();
```

Comment:
not UTC + not deterministic,

Should be replaced by a deterministic argument (in addition to lat and lon)

Reply:
now the user will have to pass a third argument for timestamp in seconds in utc along with lat and longi.